SetupScript.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Permission Denied"
```

Comment:
Other commands in this script will be able to tell the user that, and be more detailed as well. A more appropriate message I think would be "Please run as the root user"